core/deployment/src/main/java/io/quarkus/deployment/dev/DevModeMain.java

```diff
@@ -140,6 +139,18 @@ public void start() throws Exception {
         }
     }
 
+    private Path resolveProjectRoot() {
+        final Path projectRoot;
+        if (context.getProjectDir() != null) {
+            projectRoot = context.getProjectDir().toPath();
+        } else if (context.getApplicationRoot().getProjectDirectory() != null) {
+            projectRoot = Path.of(context.getApplicationRoot().getProjectDirectory());
+        } else {
+            projectRoot = new File(".").toPath();
```

Comment:
```suggestion
            projectRoot = Path.of("").normalize().toAbsolutePath();
```

Reply:
@aloubyansky I haven't changed the code for this one: 
https://github.com/quarkusio/quarkus/pull/25510/files#diff-e19b40a2e4f91d841e9af1ce689ee0ec39d1c6ce69c92823de6350e9ddd3fa9eL116

Should we change it still?

Reply:
It's pretty much equivalent, except it translates to an absolute path which, imo, should be preferred. Although given it's worked so far, you can leave it as it is, not a big deal in this case.